pkg/cmd/mom/input_resources_command.go

```diff
@@ -0,0 +1,82 @@
+
+import (
+	"context"
+
+	"github.com/openshift/multi-operator-manager/pkg/library/libraryinputresources"
+	"github.com/spf13/cobra"
+	"k8s.io/cli-runtime/pkg/genericiooptions"
+)
+
+func NewInputResourcesCommand(streams genericiooptions.IOStreams) *cobra.Command {
+	return libraryinputresources.NewInputResourcesCommand(RunInputResources, streams)
+}
+
+func RunInputResources(ctx context.Context) (*libraryinputresources.InputResources, error) {
+	return &libraryinputresources.InputResources{
+		ApplyConfigurationResources: libraryinputresources.ResourceList{
+			ExactResources: []libraryinputresources.ExactResourceID{
+				// Operator
+				libraryinputresources.ExactLowLevelOperator("authentications"),
+				// Config
+				libraryinputresources.ExactConfigResource("apiservers"),
+				libraryinputresources.ExactConfigResource("infrastructures"),
+				libraryinputresources.ExactConfigResource("oauths"),
+				libraryinputresources.ExactConfigResource("ingresses"),
+				libraryinputresources.ExactConfigResource("consoles"),
+				libraryinputresources.ExactConfigResource("proxies"),
+				libraryinputresources.ExactResource("config.openshift.io", "v1", "clusterversions", "", "version"),
+				// OpenShiftx
+				libraryinputresources.ExactResource("route.openshift.io", "v1", "routes", "openshift-authentication", "oauth-openshift"),
+				// Core
+				libraryinputresources.ExactResource("", "v1", "endpoints", "openshift-authentication", "oauth-openshift"),
+				libraryinputresources.ExactResource("", "v1", "endpoints", "default", "kubernetes"),
+				libraryinputresources.ExactResource("", "v1", "services", "default", "kubernetes"),
+				libraryinputresources.ExactResource("", "v1", "services", "openshift-authentication", "oauth-openshift"),
+				// Configuration
+				libraryinputresources.ExactConfigMap("openshift-config-managed", "oauth-openshift"),
+				libraryinputresources.ExactConfigMap("openshift-config-managed", "router-certs"),
+				libraryinputresources.ExactConfigMap("openshift-config-managed", "default-ingress-cert"),
+				// Operator
+				libraryinputresources.ExactConfigMap("openshift-authentication-operator", "trusted-ca-bundle"),
+				// Operand
+				libraryinputresources.ExactConfigMap("openshift-authentication", "v4-0-config-system-metadata"),
+				libraryinputresources.ExactConfigMap("openshift-authentication", "v4-0-config-system-service-ca"),
+				libraryinputresources.ExactSecret("openshift-authentication", "v4-0-config-system-serving-cert"),
+				libraryinputresources.ExactSecret("openshift-authentication", "v4-0-config-system-session"),
+				libraryinputresources.ExactSecret("openshift-authentication", "v4-0-config-system-router-certs"),
+				libraryinputresources.ExactSecret("openshift-authentication", "v4-0-config-system-custom-router-certs"),
+				libraryinputresources.ExactSecret("openshift-authentication", "v4-0-config-system-ocp-branding-template"),
+			},
+		},
+		OperandResources: libraryinputresources.OperandResourceList{
```

Comment:
@deads2k what is the purpose of this field?

Reply:
As we discussed in our meeting, removed